### DISCUSSION TITLE
A big thank you

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#8878] [appreciation]
This is, for me, the best developer team ever existed.
Not only you guys are always willing to hear what people have to say or sugest, but also you bring the best to the game.
Now, Monika is finally warm with her jacket and scarf, and her new sweater.
Thank you, you guys will have a special place in my heart.
Keep up the awesome work you do. <3

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
